.github/renovate.json

```diff
@@ -16,6 +16,17 @@
     }
   ],
   "packageRules": [
+    {
```

Comment:
These changes update the AKS-GPU container images for pre-release/patch versions but don't create PRs for minor version updates. Any suggestions on how to fix that?

Reply:
Can you link your fork PR which updates the minor?

Reply:
This is the current fork with the patch version updates: https://github.com/ganeshkumarashok/AgentBaker/blob/master/.github/renovate.json 

For minor versions I just set `enable` to `true` for docker format: https://github.com/ganeshkumarashok/AgentBaker/commit/c29f78597da528b96d04f6c2d58df485481e4a75

Reply:
hmmm. No idea yet. If it's still not working, maybe remove "major" and "patch" from "matchUpdateTypes" to see if minor works. Make sure you really have that updated minor version in your container registry? You can search that version in Renovate debug logs directly. If Renovate really finds it, you should be able to see some logs there.

Reply:
It works in the fork now! Had to move the aks-gpu packageRule to the bottom (as the enablement for minor versions is an exception to the broader rule), and set `ignoreUnstable` to false https://github.com/ganeshkumarashok/AgentBaker/commit/4090b89f581ca03f063dfed8815889113437c5ef